products/BellHybrid/assets_images.json

```diff
@@ -0,0 +1,216 @@
+{
+    "assets": [
+        {"name": "release2.tgz", "tarfile" :"image/assets/images/music-player/mp_note_W_G.vpi", "output": "assets/images/music-player/mp_note_W_G.vpi"},
```

Comment:
Is it a tarfile or just an input file?

Reply:
it's the filename in the tar file. As this script uses `name` as input name - which is `release2.tgz` here, and output would be `mp_note_W_G.vpi` we have to know what's the path of this file in the `name` file :)

Reply:
That does make sense ;) How about this? 
```suggestion
        {"tarfile": "release2.tgz", "input" :"image/assets/images/music-player/mp_note_W_G.vpi", "output": "assets/images/music-player/mp_note_W_G.vpi"},
```

Reply:
Feel free to close this conversation ;)

Reply:
sounds leigt, I would have to have more complicated json parsing and it was meant to be simple :) ( btw. if we drop `tarfile` which is a bad name, then have `name` + `input` -> `output` this should be ok? )